src/looper/index.ts

```diff
@@ -122,6 +124,9 @@ export function isIDValid(config: Config, ids: string): boolean {
 }
 
 export async function loopCheckSTUNServer(interval: number): Promise<any> {
+  // if stun is disabled, no need to create interval
+  if (interval === -1) return
```

Comment:
isn't this the only change needed to make the `-s -1` flag works? The other changes should be in different PR.

Reply:
another change required in loopProbe() related to looping check when STUN is connected:
now its:    (isConnectedToSTUNServer && !isPaused) ||  flags.stun === DISABLE_STUN